bin/tv-train

```diff
@@ -76,32 +80,33 @@ def initialize_training_folder(hypes, train_dir):
 
     # TODO: read more about loggers and make file logging neater.
 
-    hypes_file = tf.app.flags.FLAGS.hypes
-    _copy_parameters_to_traindir(hypes_file, "hypes.json", target_dir)
+    hypes_file = os.path.basename(tf.app.flags.FLAGS.hypes)
+    _copy_parameters_to_traindir(
+        hypes, hypes_file, "hypes.json", target_dir)
     _copy_parameters_to_traindir(
-        hypes['model']['input_file'], "data_input.py", target_dir)
+        hypes, hypes['model']['input_file'], "data_input.py", target_dir)
```

Comment:
`hypes['model']['input_file'], hypes['model']['architecture_file'], hypes['model']['objective_file']` and `hypes['model']['optimizer_file']` always exist in `hypes`. We can therefor only pass `hypes`.